pages/docs/some-react-code/index.jsx

```diff
@@ -3,18 +3,18 @@ import Demo from './_Demo'
 import DocumentTitle from 'react-document-title'
 import { config } from 'config'
 
+exports.data = {
+  title: 'Some React Code',
+  date: `${new Date()}`
```

Comment:
(I'm the OP referred to above) 

I don't need the date, I was pseudo-copying from some other example. 

As mentioned in the aforementioned thread, this is the code I have running locally, which is not working.